sound/soc/codecs/max98373-sdw.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: GPL-2.0
+ *  Copyright (c) 2020 Maxim Integrated
+ */
+
+#ifndef _MAX98373_SDW_H
+#define _MAX98373_SDW_H
+
+#include "max98373.h"
+
+struct sdw_stream_data {
+	struct sdw_stream_runtime *sdw_stream;
+};
+
+struct max98373_sdw_priv {
+	struct regmap *regmap;
+	int reset_gpio;
+	unsigned int v_slot;
+	unsigned int i_slot;
+	unsigned int spkfb_slot;
+	bool interleave_mode;
+	unsigned int ch_size;
+	bool tdm_mode;
+	struct sdw_slave *slave;
+};
```

Comment:
are these structures used outside of the driver? I see other codecs drivers do that too, but you could also just define them in the driver

Reply:
OK. Moved definition to the driver.